debian-8.8-i386.json

```diff
@@ -221,25 +221,25 @@
     }
   ],
   "variables": {
-    "box_basename": "debian-8.7-i386",
+    "box_basename": "debian-8.8-i386",
     "build_timestamp": "{{isotime \"20060102150405\"}}",
     "cpus": "1",
     "disk_size": "40960",
     "git_revision": "__unknown_git_revision__",
     "headless": "",
     "http_proxy": "{{env `http_proxy`}}",
     "https_proxy": "{{env `https_proxy`}}",
-    "iso_checksum": "f3debfa96a024b9c1a4b342db96c6afcd0e9adb94a6144ed99e34bce110be2d5",
+    "iso_checksum": "0834db8a6c6eab749be5d25b0451fa46ea96dfb91134df8f26323ff8b79b01ca",
     "iso_checksum_type": "sha256",
-    "iso_name": "debian-8.7.1-i386-CD-1.iso",
+    "iso_name": "debian-8.8.0-i386-CD-1.iso",
     "memory": "512",
     "metadata": "floppy/dummy_metadata.json",
     "mirror": "http://cdimage.debian.org/cdimage/release",
```

Comment:
These URLs need to get updated now that 9 is out. This should be http://cdimage.debian.org/cdimage/archive